worlds/ff1/Items.py

```diff
@@ -39,13 +41,13 @@ class FF1Items:
     def _populate_item_table_from_data(self):
         base_path = Path(__file__).parent
         file_path = (base_path / "data/items.json").resolve()
-        with open(file_path) as file:
-            items = json.load(file)
-            # Hardcode progression and categories for now
-            self._item_table = [ItemData(name, code, "FF1Item", ItemClassification.progression if name in
-                                FF1_PROGRESSION_LIST else ItemClassification.useful if name in FF1_USEFUL_LIST else
-                                ItemClassification.filler) for name, code in items.items()]
-            self._item_table_lookup = {item.name: item for item in self._item_table}
+        file = pkgutil.get_data(__name__, os.path.join("data", "items.json")).decode("utf-8")
```

Comment:
```suggestion
        file = pkgutil.get_data(__name__, "data/items.json").decode("utf-8")
```
ref https://github.com/ArchipelagoMW/Archipelago/pull/4232
for this and the other similar suggestion (also there's no need for the `os` import after this change)